addons/l10n_in_ewaybill_stock/models/stock_move.py

```diff
@@ -28,20 +28,22 @@ class StockMove(models.Model):
 
     @api.depends('product_id.uom_id', 'product_id.standard_price', 'l10n_in_ewaybill_id')
     def _compute_l10n_in_ewaybill_price_unit(self):
-        for line in self.filtered(lambda line: line.l10n_in_ewaybill_id.state not in ['generated', 'cancel']):
```

Comment:
@jco-odoo @auma-odoo the issue lays here whenever there is no e-waybill it used to consider the state False.
Tho the condition was kept for to pre-compute the ewaybill_price_unit when until the ewaybill was not saved but nwo it really seems like disaster.

As of this fix would work well.

@jco-odoo But we need to think other way to get it compute rightly done for Ewaybill stock